jobs/v3/auto_complete_sample.go

```diff
@@ -0,0 +1,114 @@
+package cjdsample
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	talent "google.golang.org/api/jobs/v3"
+)
+
+// [START auto_complete_job_title]
+
+/**
+ * Auto completes job titles within given companyName
+ */
+func JobTitleAutoComplete(service *talent.Service, companyName string, query string) (*talent.CompleteQueryResponse, error) {
+	complete := service.Projects.Complete(GetParent()).Query(query).LanguageCode("en-US").Type("JOB_TITLE").PageSize(10)
+	if companyName != "" {
+		complete.CompanyName(companyName)
+	}
+	resp, err := complete.Do()
+	if err != nil {
+		log.Fatalf("Failed to auto complete with query %s in company %s, Err: %v", query, companyName, err)
```

Comment:
I think it's better to stop the run-through tests when error occurs because in most run()/entry() the following cases will fail too, that's why I'm using log.Fatalf().

Reply:
The snippets should be for how users would use them. Returning errors is normal and idiomatic. Calling `log.Fatal` when the function can return the error is not recommended.